packages/popover/src/vaadin-popover.d.ts

```diff
@@ -109,6 +109,12 @@ declare class Popover extends PopoverPositionMixin(
    */
   accessibleNameRef: string | null | undefined;
 
+  /**
+   * Element on which `aria-haspopup`, `aria-expanded` and `aria-controls` attributes
+   * will be set depending on the popover opened state. Defaults to `target`.
+   */
+  ariaTarget: HTMLElement | undefined;
```

Comment:
Do you need to set this property manually for every popover that is on an input field? Wouldn't it be easier if every input field would define what its aria target is, and popover would then automatically use something like `target.ariaTarget || target`? Or maybe something more specific like `popoverAriaTarget` if needed.

Reply:
Good point, thanks. The field components do have `ariaTarget` already, so I can probably use that.
Then we could also backport this as it would mean no need to add a dedicated API for this.

Reply:
Updated the PR accordingly and renamed to `fix:` instead of `feat:`.